src/localization/localization.ts

```diff
@@ -128,7 +129,19 @@ const localization = {
             join: "Join",
             connected: (count: number) => `Connected to ${pluralize(count, "Zeplin Component")}.`,
             connect: "Connect to Zeplin components",
-            notFound: (name: string) => `No component matching “${name}” in projects/styleguides you’re a member of`,
+            askMigration: "Would you like to migrate zeplinNames on your configuration to zeplinIds?",
+            migrate: "Migrate Zeplin component names to ids",
+            migrationInProgress: "Getting components data for migration…",
+            noBarrelFoundForMigration: "No project or styleguide found on configuration file.",
+            noComponentFoundForMigration: "No component found on configuration file.",
+            noNameFoundForMigration: "No Zeplin component name found on configuration file.",
+            barrelErrorForMigration: "Zeplin components data could not be fetched. Please check your projects and styleguides on configuration file.",
+            noZeplinComponentFoundOnBarrelsForMigration: "No Zeplin components found on your projects and styleguides. Please check you added correct projects and styleguides to configuration file.",
+            migrated: (count: number) => `Migrated ${pluralize(count, "Zeplin component name")}.`,
+            migrationErrors: (count: number) => `${pluralize(count, "Zeplin component name")} can not be migrated because matching Zeplin component ids can not be found. Please check their names are correct.`,
```

Comment:
```suggestion
            migrationErrors: (count: number) => `${pluralize(count, "Zeplin component name")} can not be migrated because matching Zeplin component IDs can not be found. Please check whether component names still exist on Zeplin.`,
```

Reply:
👍 except "IDs" part